CONTRIBUTING.md

```diff
@@ -0,0 +1,260 @@
+and _drafts_, the library's maintainers may request that these new features are
+disabled by default with opt-in option flags or they may simply wait to merge a
+well-tested pull request until the RFC is _released_.
+
+## Guiding Principles
+
+HMIS API's evolution is guided by a few principles. Suggested contributions
+should use these principles to guide the details of an RFC and decisions to move
+forward.
+
+- **User driven**
+  The HMIS API specification and its libraries are built for the purpose of reducing 
+  homelessness. Future changes to the API specification must be evaluated under this 
+  lense first and with the greatest weight.
+
+- **Backwards compatibility**
+
+  HMIS API libraries should strive to maintain the structure and meaning of existing
+  resources. Future changes should avoid incompatability with the existing
+  resources and should avoid breaking changes across prior verions of this 
+  specification. When breaking changes are required they should be clearly tied to
+  the guiding principles here and follow the guidelines for 
+  (schema evolution)[SCHEMA-EVOLUTION].
+
+- **Favor no change**
+
+  As the HMIS API is implemented under the collaboration of hundreds of individuals, 
+  incorporating any change has a high cost. Proposed changes must meet a very high bar 
+  of added value. The burden of proof is on the contributor to illustrate this value.
+
+- **Enable new capabilities motivated by real use cases**
+
+  Every change should intend on unlocking a real and reasonable use case. Real
+  examples are always more compelling than theoretical ones, and common
+  scenarios are more compelling than rare ones. RFCs should do more than offer a
+  different way to reach an already achievable outcome.
+
+- **Simplicity and consistency over expressiveness and terseness**
+
+  There are many avenues for data exchange and interoperability between HMIS and
+  other systems within and outside of Communities of Care. This specification seeks
+  to enable a broad spectrum of those use cases in a way that can be applied across
+  those use cases. "Possible but awkward" is often favored over more complex or 
+  scenario-specific alternatives. Simplicity (e.g. fewer concepts) is more important 
+  than expressing more sophisticated ideas or writing less.
+
+- **Preserve option value**
+
+  It's hard to know what the future brings; whenever possible, decisions should
+  be made that allow for more options in the future. Sometimes this is
+  unintuitive: specification rules often begin more strict than necessary with a 
+  future option to loosen when motivated by a real use case.
+
+- **Understandability is just as important as correctness**
+
+  The HMIS API specification, despite describing technical behavior, is intended to be
+  read by people. Use natural tone and include motivation and examples.
+
+## RFC Contribution Champions
+
+Contributing to GraphQL requires a lot of dedicated work. To set clear
```

Comment:
I was under the impression that we are working with a JSON REST API format and not a GraphQL. Confirming that you meant GraphQL and not REST API.

Reply:
dun